features/dedicatedip/src/main/java/com/kape/dedicatedip/ui/screens/mobile/SignupDedicatedIpPurchaseSuccessScreen.kt

```diff
@@ -65,7 +65,7 @@ fun SignupDedicatedIpPurchaseSuccessScreen() = Screen {
         )
         Spacer(modifier = Modifier.weight(1.0f))
         PrimaryButton(
-            text = stringResource(id = R.string.dip_signup_generate_your_token),
+            text = stringResource(id = R.string.logjn_continue),
```

Comment:
is that a `j` instead of `i`?

Reply:
yup, a typo from when we introduced that key. Fixing it is a bit complex as we need to update all languages and the sources in our translation dashboard so that they are not picked up again for translation